src/plugins/enum/README.md

```diff
@@ -0,0 +1,19 @@
+- infos = Information about the enum plugin is in keys below
+- infos/author = Thomas Waser <>
+- infos/licence = BSD
+- infos/needs =
+- infos/provides = check
+- infos/placements = presetstorage
+- infos/description =
+
+## Introduction ##
+
+The Enum plugin checks string values of Keys by comparing it against a list of valid values.
+
+## Usage ##
+
+The plugin checks every Key in the Keyset for the Metakey `check/enum` containing a list with the syntax `['string1', 'string2', 'string3', ..., 'stringN']` and compares each value with the string value of the Key. If no match is found an error is returned.
```

Comment:
Thinking about it, the `[]` are not even needed when refering to outside defined enums is allowed. You simply know if you refer to something else, when its not quoted in `'`.

So given a defined `abc = 'a', 'b', 'c'` one can use `check/type` with `abc, 'd'` without ambiguity. What do you think?